### ISSUE TITLE
AdGuard DNS Server being routed to Singapore from Canada with AdGuard VPN

### ISSUE THREAD

**Issue by Batman2814** [AdguardTeam/AdGuardDNS#718]
### Platform

Windows

### Protocol

DNS-over-QUIC

### Do you use AdGuard app?

Yes I am

### Traceroute to AdGuard DNS

![20231223_17300302_Edited](https://github.com/AdguardTeam/AdGuardDNS/assets/96619913/ef214a93-2238-47a3-a838-3727fb577cf5)
![20231223_17301390_Edited](https://github.com/AdguardTeam/AdGuardDNS/assets/96619913/5f19e809-3365-4b6a-a69b-1f18823ebcea)

### Issue Details

NA

### Expected Behavior

The server should be much closer, like the New York server for AdGuard DNS.  

### Actual Behavior

Location is being routed all the way across the world from where I am. 

### Screenshots

![20231223_16354739_Edited](https://github.com/AdguardTeam/AdGuardDNS/assets/96619913/80db622f-f5ec-49d6-b028-bae1d5250b08)
![20231223_16382529_Edited](https://github.com/AdguardTeam/AdGuardDNS/assets/96619913/89fe58d5-f6aa-4191-a849-3073078e9a83)

**Comment by Batman2814** [comment]
![20231223_18223293_Edited](https://github.com/AdguardTeam/AdGuardDNS/assets/96619913/9bddd7ed-65bc-4f2c-9d75-ad48689397b5)

**Comment by Batman2814** [comment]
![20231223_18223293_Edited](https://github.com/AdguardTeam/AdGuardDNS/assets/96619913/9139f1dd-7fba-4ea7-9f17-f61d2fcc155d)

**Comment by joetimmy** [comment]
yep there is alot of DNS hop issues, currently in Toronto its routing to chicago adding 20+Ms on delays, no ideal. 
@adguard would be nice to address all your canadian routing issues asap by moving all the new ips to local servers.

**Comment by gpodder770** [comment]
> yep there is alot of DNS hop issues, currently in Toronto its routing to chicago adding 20+Ms on delays, no ideal. @adguard would be nice to address all your canadian routing issues asap by moving all the new ips to local servers.

The canadian problem seems to be still here
At home I use Bell and dns connects to chicago

at university of winnipeg it connects to new york

but for nextdns it always connects to torronto server and same is for controld


Don’t know what the heck is wrong with Adguard

**Comment by Mr-McMuffin** [comment]
I dont have the issue anymore, mine connects to Toronto - try to use a new server and delete your old one.

**Comment by gpodder770** [comment]
> I dont have the issue anymore, mine connects to Toronto - try to use a new server and delete your old one.

still same :( 
![image](https://github.com/user-attachments/assets/ff3eeb89-2399-4899-982d-4657e4da0fd6)

**Comment by Chinaski1** [comment]
Hello there!

AdGuard DNS uses [Anycast](https://adguard-dns.io/en/blog/anycast-and-bgp-how-adguard-dns-serves-millions-of-queries.html) technology so we can't manage traffic routing 100% of the time.

At this point, we're compiling problems in internal list with the possibility of their correction in nearest future.
Please provide following information via email to support@adguard.com (also share a link to this issue on GitHub):

1) Your IP address: https://adguard.com/en/test.html
2) ASN. 
3) Provider name.
4) DNS server location: https://adguard.com/en/test.html

I am closing this task as it goes to the internal list of routing issues.
